### PR TITLE
RavenDB-14729 & RavenDB-14800

### PR DESCRIPTION
**Pull request by aviv86**
https://issues.hibernatingrhinos.com/issue/RavenDB-14729
https://issues.hibernatingrhinos.com/issue/RavenDB-14800

### CODE REVIEW

File: src/Raven.Server/Documents/Queries/AST/Query.cs [hunk 1]
```diff
@@ -447,20 +448,20 @@ public IEnumerable<object> GetFinalValues()
...
-                        _values[i] = (double)_values[i] / (long)_count[i];
+                        _values[i] = _values[i] / _count[i];
                     }
                     break;
                 default:
                     throw new ArgumentOutOfRangeException("Unknown aggregation operation: " + Aggregation);
             }
 
-            return _values;
+            return _values.Cast<object>();
```
Comment (karmeli87) [inline src/Raven.Server/Documents/Queries/AST/Query.cs:464]: I think those casting will box.. is it possible to get rid of the `object` and use `long` or `double` instead?

Comment (aviv86) [reply src/Raven.Server/Documents/Queries/AST/Query.cs:464]: we use those aggragated values in `AddTimeSeriesResult` :  https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Server/Documents/Queries/Results/TimeSeriesRetriever.cs#L828 
we turn them into `DynamicJsonArray`..  and `DynamicJsonArray`  only works with objects

**Review by karmeli87 — APPROVED** [review]

**Review by ppekrol — APPROVED** [review]
@aviv86 can you check failing tests?

**Comment by karmeli87** [comment]
@ppekrol failing tests are not related to this PR.

**Pull request merged**